src/preact/base-element.js

```diff
@@ -725,6 +728,21 @@ function collectProps(Ctor, element, ref, defaultProps) {
   return props;
 }
 
+/**
+ * Clones an Element into a VNode.
+ * @param {!Element} element
+ * @return {!PreactDef.Renderable}
+ */
+function shallowCloneVNode(element) {
+  const {attributes} = element;
+  const props = {key: element};
```

Comment:
Strangely, React doesn't allow objects for this. Preact seems fine.

Reply:
Since this is the AMP layer, I'm assuming this is ok to keep?

Reply:
Yup.